insights/specs/datasources/yum_updates.py

```diff
@@ -21,8 +22,9 @@ def sorted_cmp(it, cmp):
 
 class DnfManager:
     """ Performs package resolution on dnf based systems """
-    def __init__(self):
+    def __init__(self, build_pkgcache=False):
         self.base = dnf.base.Base()
+        self.build_pkgcache = build_pkgcache
```

Comment:
what about setting it directly to `self.base.conf.cacheonly = !build_pkgcache` here?